### DISCUSSION TITLE
404 for `https://repos.r2.relay.so/rpm/el.repo/repodata/repomd.xml`

### DISCUSSION THREAD

**Discussion by Blue-Nomad** [cachewerk/relay#125] [Support] [answered]
I've just installed the repo and I'm getting this error when running ```dnf up```

Any help is appreciated!

**Comment by tillkruss** [comment, marked as answer]
That looks like an invalid URL because of a misconfiguration. The URL should be any of these:

```
https://repos.r2.relay.so/rpm/el{VERSION}/repodata/repomd.xml

https://repos.r2.relay.so/rpm/el9/repodata/repomd.xml
```

What does your `/etc/yum.repos.d/cachewerk.repo` look like? 

What's you're OS and version?

**Reply by Blue-Nomad** [reply]
Ah thanks, I'm using Ansible and it asked for the baseurl and I used https://repos.r2.relay.so/rpm/el.repo from the command in the docs. I've replaced that with https://repos.r2.relay.so/rpm/el$releasever/ and it's now working.
